File: WebApplication1/Models/AuditoriumType.cs
using System.ComponentModel.DataAnnotations;

namespace WebApplication1.Models
{
	/// <summary>
	/// Тип аудитории
	/// </summary>
	public class AuditoriumType
	{
		public int Id { get; set; }

		[Display(Name = "Аббревиатура")]
		public string AuditoriumAbbreviation { get; set; }

		[Display(Name = "Полное наименование")]
		public string AuditoriumName { get; set; }
	}
}

File: WebApplication1/ClientApp/app/auditoriumType/auditoriumtype.ts
export class AuditoriumType {
    constructor(
        id?: number,
        auditoriumAbbreviation?: string,
        auditoriumName?: string) {
        this.id = id;
        this.auditoriumName = auditoriumName;
        this.auditoriumAbbreviation = auditoriumAbbreviation;
    }
    public id: number;
    public auditoriumAbbreviation: string;
    public auditoriumName: string;
}